.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     # List of available Android Docker images: https://circleci.com/developer/images/image/cimg/android#image-tags
     executor:
       name: android/android-docker
-      tag: 2022.09.2-node
+      tag: 2024.01.1-node
```

Comment:
What's difference between 2024.01.1-node and 2024.01.1?
Other extensions are using 2024.01.1

Reply:
`-node` includes Node.js in the image, which is required when using Spotless + Prettier.